docs/getting_started/requirements.md

```diff
@@ -33,7 +33,7 @@ event MPM is recommended, if you need to use prefork you also need the `mod_php`
 
 - MariaDB 10.3+
 - MySQL 8.0
-- PostgreSQL 10+
+- PostgreSQL 14
```

Comment:
As mentioned internally, I think it would be safer for 3.3 to explicitly recommend it instead of requiring (unless this section is all about recommendation?). Do we intentionally skip `+` sign?